### PR TITLE
test_1 added

### PR DESCRIPTION
**Pull request by TarasSkip**

### CODE REVIEW

File: pages/login.page.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ let WebInput = require('../elements/input');
 let loginFormEmailLocator = by.name('login');
 let loginFormPasswordLocator = by.name('password');
 let loginButtonLocator = by.css('input[type="submit"]');
-
+let registrationLinkLocator = by.css('body > div.wrapper > div.content.row > div > div.viewbox > div.text > p > a');
```
Comment (nbabii) [inline pages/login.page.js]: Please optimise locator

File: pages/registration.page.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+let BasePage = require('./base.page');
+let WebInput = require('../elements/input');
+
+let registrationFormEmailLocator = by.name('email');
+let registrationFormPasswordLocator = by.name('password');
+let registrationFormNicknameLocator = by.name('login');
+let registrationButtonLocator = by.css('input[type="submit"]');
+let registrationTextLocator = by.css('body > div.wrapper > div.content.row > div > div > div > div.cell-4.cell-md > p');
```
Comment (nbabii) [inline pages/registration.page.js]: same here

File: pages/registration.page.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    getRegistrationFormNicknameElement() {
+        return new WebInput(element(registrationFormNicknameLocator), "Nickname input");
+    }
+
+    getRegistrationButtonElement() {
+        return new WebInput(element(registrationButtonLocator), "Registration button");
+    }
+
+    getRegistrationTextElement() {
+        return element(registrationTextLocator);
```
Comment (nbabii) [inline pages/registration.page.js]: You need to use some element wrapper here

**Review by nbabii — CHANGES_REQUESTED** [review]

File: pages/login.page.js (hunk 1 above)
Comment (TarasSkip) [reply pages/login.page.js]: Changed on "_let registrationLinkLocator = by.css('a[href*="register"]');_"

File: pages/registration.page.js (hunk 2 above)
Comment (TarasSkip) [reply pages/registration.page.js]: Changed on _"let registrationTextLocator = by.css('.h3');"_

File: pages/registration.page.js (hunk 3 above)
Comment (TarasSkip) [reply pages/registration.page.js]: Added _text.js_ file

File: elements/text.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+let BaseElement = require('./base.element');
+
+class TextBox extends BaseElement {
+    async getText(text) {
```
Comment (nbabii) [inline elements/text.js]: no need in `text` argument here

File: elements/text.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+let BaseElement = require('./base.element');
```
Comment (nbabii) [inline elements/text.js]: Rename file to textBox.js to make it more clear.

**Review by nbabii — CHANGES_REQUESTED** [review]

File: elements/text.js (hunk 4 above)
Comment (TarasSkip) [reply elements/text.js]: Done

File: elements/text.js (hunk 5 above)
Comment (TarasSkip) [reply elements/text.js]: Done

**Pull request merged**